### PR TITLE
style: mui theme switch

### PR DESCRIPTION
**Pull request by goldsteinsveta**
@Stefie I've added a theme switch and replaced `styled` of `styled-components` to native MUI styling methods in places

### CODE REVIEW

File: projects/burnr/src/themes/substrate/light.js [hunk 1]
```diff
@@ -1,184 +0,0 @@
-import { createMuiTheme }  from '@material-ui/core/styles'
```
Comment (Stefie) [inline projects/burnr/src/themes/substrate/light.js:1]: I think you can also delete `projects/burnr/src/themes/substrate/dark.js`. It's not in use anymore, right?

File: projects/burnr/src/themes/substrate/dark.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { ThemeOptions } from '@material-ui/core/styles';
+
+const dark: ThemeOptions = {
+	palette: {
```
Comment (Stefie) [inline projects/burnr/src/themes/substrate/dark.ts:4]: I'd keep the primary & secondary theme colors from the old json files here (projects/burnr/src/themes/substrate/light.js) and also use the `colors.js` file.

That way you just need to change a color in the colors.js file once and it applies to both themes.

I took them from the brand deck. It's the pink and the cyan

https://github.com/paritytech/substrate-connect/pull/7/files#diff-2f0b32fbdf8fe51d80c3bcb4dd81d352L34
```
  primary: {
    light: colors.cyan.light,
    main: colors.cyan.main,
    dark: colors.cyan.dark,
    contrastText: colors.black
  },
  secondary: {
    light: colors.pink.light,
    main: colors.pink.main,
    dark: colors.pink.dark,
    contrastText: colors.white
  },
```

File: projects/burnr/src/themes/substrate/light.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { ThemeOptions } from '@material-ui/core/styles';
+
+const light: ThemeOptions = {
+	palette: {
+		type: 'light',
+		background: {
+			default: '#F7F7F7',
+		},
+		primary: {
```
Comment (Stefie) [inline projects/burnr/src/themes/substrate/light.ts:9]: Same comment as in the dark.js.

File: projects/burnr/src/themes/substrate/light.js (hunk 1 above)
Comment (goldsteinsveta) [reply projects/burnr/src/themes/substrate/light.js:1]: done

File: projects/burnr/src/themes/substrate/dark.ts (hunk 2 above)
Comment (goldsteinsveta) [reply projects/burnr/src/themes/substrate/dark.ts:4]: will restore and get back!

**Pull request merged**